airflow/executors/base_executor.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, parallelism: int = PARALLELISM):
         self.running: Set[TaskInstanceKey] = set()
         self.event_buffer: Dict[TaskInstanceKey, EventBufferValueType] = {}
 
-    def start(self):  # pragma: no cover
+    def start(self, job_id=None):  # pragma: no cover
```

Comment:
given that the `start` interface is changed, we also need to update other executors, like https://github.com/apache/airflow/blob/master/airflow/executors/celery_kubernetes_executor.py#L60-L63